package.json

```diff
@@ -9,12 +9,6 @@
     "email": "951711127@qq.com",
     "url": "https://github.com/js-sdsl/js-sdsl"
   },
-  "browserslist": [
```

Comment:
Please remain browserslist because I'm not sure when the other PR will be merged.

Reply:
If you don't clear the browser list, it won't build. 

This is because the `rollup-plugin-ts` plugin prioritizes the information in the browserlist over the typescript configuration to determine the JavaScript version of the build target.